packages/editor/src/components/entities-saved-states/index.js

```diff
@@ -132,4 +137,6 @@ function EntitiesSavedStates( { isOpen, close } ) {
 	) : null;
 }
 
-export default withFocusReturn( EntitiesSavedStates );
+export default compose( [ withFocusReturn, withConstrainedTabbing ] )(
```

Comment:
Should this be considered a "dialog" and use the new useDialog experimental react hook, it does a bit more than these two HoCs (focus return, focus on mount, focus outside, constrained tabbing)? I'm not sure just asking :)

Reply:
Yes, I get what you mean, I'm not sure either. Anyhow, I'm just providing a fix for the issue (it's on the 5.8 FSE must-have list). 

Might be something for a separate issue? Maybe it's already being tracked somewhere :)

Reply:
I am not sure either, maybe @joedolson @alexstine @diegohaz has thoughts about this?

Reply:
It sounds very plausible that this should be considered a dialog; but it's not totally obvious to me what the "multi-save save panel" is. A specific click path to open the panel would be helpful, possibly with a screenshot so that I can verify that I'm looking at the referenced panel.

Reply:
It's the panel that opens up when clicking _Save_ after making changes in the site editor (`wp-admin/admin.php?page=gutenberg-edit-site`).

![savepanel](https://user-images.githubusercontent.com/772137/115999996-aee56200-a5ee-11eb-93f4-61019f2d1cc9.gif)

Reply:
Thanks, @vdwijngaert - that's very helpful. In my opinion, yes, this is a good example of the dialog pattern, since it's the result of an action that then requires additional feedback to confirm and complete. You mention that the useDialog hook is experimental, and we should make sure to test it appropriately. I looked at some docs on useDialog (https://react-spectrum.adobe.com/react-aria/useDialog.html), and I can't see any indication that it's experimental, but I'm not sure how that information would be exposed.

Reply:
I think @youknowriad was talking about the internal [useDialog](https://github.com/WordPress/gutenberg/tree/trunk/packages/compose/src/hooks/use-dialog) hook. I agree this should be refactored to using the new useDialog hook. I think it's best to discuss adoption of `useDialog` further in a seperate issue. After all, more dialogs, including the pre publish panel can benefit from this new hook. 

Anyhow, this PR fixes the issue of focus getting lost, because tabbing was not constrained. LGTM :)

Reply:
I think if useDialog creates the desired behavior, we should just go ahead and use it as we're slowly and surely consolidating all of the a11y behaviors using these hooks.

Reply:
Also works for me. In this PR?

Reply:
yeah, that works

Reply:
Done.